### DISCUSSION TITLE
Drops downs with the tab index update seem to get obstructed now, how to fix?

### DISCUSSION THREAD

**Discussion by tayywayne** [saadeghi/daisyui#1906] [Q&A] [answered]
I am working on a full stack app and currently working on building out the front end with React.
In this new 3.0 update, a few components seem to have broken in the way i was using them.
I was utilizing a few of the dropdowns throughout my project and the nav bar, and even after converting any classes from a button to the details tag or using the tabIndex={0}, both seem to not be fixing the problem.
**example of the problem i have :**
![Screenshot_155](https://github.com/saadeghi/daisyui/assets/101381253/2a173c21-54fb-4dcd-8706-4a3ae4b01827)
![Screenshot_156](https://github.com/saadeghi/daisyui/assets/101381253/b9b4f85f-5852-4b5c-b59c-e896f94d4a6a)
it seems now this dropdown show get obstructed by whatever might be in front of it. i have only seen this with dropdowns and not Modals.

**related code**
_for the nav dropdown_
`    <div className="dropdown dropdown-end">
      <label tabIndex={0} className="btn btn-ghost rounded-btn">
        Click
      </label>
      <ul
        tabIndex={0}
        className="dropdown-content menu p-2 shadow bg-neutral rounded-box w-52"
      >
        <li>
          <a>Item 1</a>
        </li>
        <li>
          <a>Item 2</a>
        </li>
      </ul>
    </div> `

_for the member card dropdown_
`          <div className="dropdown">
            <label tabIndex={0} className="link link-hover font-bold m-1">
              {member.first_name}
            </label>
            <div
              tabIndex={0}
              className="dropdown-content card card-compact w-64 p-2 shadow bg-neutral text-primary-content"
            >
              <div className="card-body">
                <h3 className="card-title">{member.first_name} LAST NAME</h3>
                member bio goes here
                <a className="link link-hover font-bold">View Full Profile</a>
              </div>
            </div>
          </div> `
          
ideally i would not like to use the details version (even though that one has the same problem right now) as i dont want to have to click on the button again to close the dropdown. 
I have tried to mess with the tab index number, searched online for people with similar problems but not avail.
for front end styling i am right now only using tailwind classes and daisyUI components.
is there maybe a class from just tailwind that could help with this and overwrite my problem with daisyUI?

_any ideas or help appreciated very much <3_

**Comment by polyzen** [comment, marked as answer]
https://daisyui.com/docs/changelog/#breaking-changes

```
dropdown doesn’t have a z-index by default now.

Using z-index for dropdown as a default style was a mistake because sometimes we need elements on top of th dropdown and sometimes we want the dropdown to be on top of other elements. Now you can use z-[1] or any other z-index class to control it as you want.
```

**Reply by tayywayne** [reply]
I should have read the update in full or at least looked for breaking changes on the update itself. Thank you for pointing me in the right direction and teaching me a lesson!

**Reply by polyzen** [reply]
I too started trying out daisyUI 3 before finishing reading the release notes and learned this change the hard way. :p
